.azure/reusable-test.yml

```diff
@@ -100,7 +100,7 @@ jobs:
         displayName: 'Shallow Checkout Repo'
 
       - bash: |
-          choco install -y --requirechecksum=true --checksum=d58e81b96d53ded74570ad028d605fcfa1bfcc2e7cb2f5ab24bd64901b0c8783 --checksum-type=sha256 procdump --version=11.0
```

Comment:
https://learn.microsoft.com/en-us/sysinternals/downloads/procdump
says that procdump is now on 11.1.  Should we update to 11.1?
11.1 supposedly fixes some bugs: https://techcommunity.microsoft.com/blog/sysinternals-blog/procdump-v11-1/4469763